scripts/ci/create-plunker-bundle.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/*
+ * This script creates a bundle of all components and publishes it to Firebase.
+ * The bundle will be used to load a Plunker Demo of Angular Material 2.
+ */
+
+const spawn = require('child_process').spawnSync;

```

Comment:
Actually we're creating a firebase for this right now, and we'll add the login key to Travis. We'll let you know what key to use :)